rclcpp/include/rclcpp/parameter_client.hpp

```diff
@@ -136,9 +137,9 @@ class AsyncParametersClient
   std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>
   set_parameters(
     std::vector<rclcpp::parameter::ParameterVariant> parameters,
-    std::function<
-      void(std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>)
-    > callback = nullptr)
+    std::function<void(
+      std::shared_future<std::vector<rcl_interfaces::msg::SetParametersResult>>)> callback =
+    nullptr)
```

Comment:
Can you please post the exact configuration of your workspace (`vcs export --exact`) to figure out what is different because it does not alter the file for me.